base_module_record/wizard/base_module_record_data.py

```diff
@@ -73,12 +73,14 @@ def record_objects(self):
                 if "_auto" in dir(obj_pool):
                     if not obj_pool._auto:
                         continue
-            recording_data = [("query", (self.env.cr.dbname, 
-                                         self.env.user.id, obj_name, 
-                                         "copy", s_id.id, {}), 
-                               {}, s_id.id) for s_id in obj_pool.search(search_condition)]
+            search_ids = obj_pool.search(search_condition)
+            for s_id in search_ids:
+                args = (self.env.cr.dbname, self.env.user.id, obj_name,
+                        'copy', s_id.id, {})
+                recording_data.append(('query', args, {}, s_id.id))
+
```

Comment:
whats the difference? 
inline was better isnt it ?

Reply:
Yes, Inline is better but in earlier code it was fetched data only last record, So that's the reason I have improved code.

Reply:
na bane! show to Vimal.

Reply:
Done